platform/x11/os_x11.cpp

```diff
@@ -1756,6 +1756,18 @@ void OS_X11::request_attention() {
 	XFlush(x11_display);
 }
 
+void *OS_X11::get_native_handle(int p_handle_type, int p_index) {
+	/* Godot 3.2 does not have multiple window support so we ignore p_index */
```

Comment:
Then maybe `p_index` doesn't need to be added in the 3.2 version of this PR?

Reply:
Yeah I've been wondering whether we should leave that out and then just add it back in when I do the PR for Godot 4